frontend/src/pages/index.tsx

```diff
@@ -33,9 +39,53 @@ const OperationButtons: React.VFC<OperationButtonProps> = (props) => {
   );
 };
 
+interface newTaskFormValues {
+  title: string;
+  description: string;
+}
+
 const IndexPage = () => {
   const { tasks, setTasks } = useFetchTasks();
 
+  const newTaskSchema = yup.object({
+    title: yup
+      .string()
+      .max(20, "20文字以内にしてください")
+      .required("入力必須です"),
+    description: yup
+      .string()
+      .max(20, "20文字以内にしてください")
+      .required("入力必須です"),
+  });
+
+  const {
+    control,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<newTaskFormValues>({
+    resolver: yupResolver(newTaskSchema),
+    mode: "onChange",
+    defaultValues: {
+      title: "",
+      description: "",
+    },
+  });
+
+  const onSubmit = async (formValues: newTaskFormValues) => {
```

Comment:
分割代入使うとparamsに渡すときにスッキリ書けそうです！
参考：https://qiita.com/FumioNonaka/items/58358a29850afd7a0f37#%E5%88%86%E5%89%B2%E4%BB%A3%E5%85%A5%E3%82%92%E4%BD%BF%E3%81%86
```suggestion
  const onSubmit = async ({title, description}: newTaskFormValues) => {
```

Reply:
ありがとうございます！
修正しました。 https://github.com/bst-n-sugita/next-todo-app-starter/pull/4/commits/74525f993e5904fa83c66c071d38dc30076662db